gatsby-config.js

```diff
@@ -72,7 +78,7 @@ module.exports = {
                 overrides: {
                   // or disable plugins
                   inlineStyles: false,
-                  cleanupIDs: false,
+                  cleanupIds: false,
```

Comment:
I am guessing this resolves the `cleanupIds` error that we are sseing during the build, right?

Reply:
Thanks for reviewing. Yes, that is correct.

---

Possible reason why: 

The original property value came from Gatsby's [docs](https://github.com/zabute/gatsby-plugin-svgr#options).
![image](https://github.com/layer5io/layer5/assets/90356410/7c9eda80-202b-43d6-bd1a-bb940cc44265)

but then I believe the warning started appearing in build logs after this package upgrade: 
`Bump @svgr/webpack from 6.5.1 to 8.0.1` [#4467](https://github.com/layer5io/layer5/pull/4467)

and then when looking at the SVGO github's [plugin chart](https://github.com/svg/svgo#built-in-plugins), the property value had different casing:
![image](https://github.com/layer5io/layer5/assets/90356410/5d0cd496-dbc1-43da-85c3-0eefe6f31260)

After correcting the config settings with the SVGO casing the warning did not appear anymore.

Cheers!

Reply:
@randychilau @Nikhil-Ladha, we're still seeing these messages while building. `cleanupIds` is enabled by default as @randychilau pointed out. Do we know why it's necessary to disable this minification? At first blush, minification is ideal. Was having this enabled, corrupting some images?

Reply:
We should find that this mis-capitalization is once again corrected and that the warning messages disappear - https://github.com/layer5io/layer5/commit/329de49a7bc09eddce1c56f16c5f1eab1f1a8efb#diff-b5e305780d9d473da97c61beab8bc36e5e8871b360942e4686c9b20d8c5d4cfaR75

Reply:
Hi @leecalcote,

regarding the question: 
>"Do we know why it's necessary to disable this minification? At first blush, minification is ideal. Was having this enabled, corrupting some images?"

The `cleanupIds` option is disabled to prevent conflicts of multiple SVGs on the same page:
>Note: SVGO does not produce unique IDs when using the cleanupIDs option; if you’re using SVGs that rely on IDs (e.g. to target template elements with use) and include multiple SVGs on the same page you may wish to disable the cleanupIDs option to prevent conflicts. ([src](https://www.gatsbyjs.com/plugins/gatsby-plugin-svgr/))

If I recall correctly, the Layer5 site required `cleanupIds` to be `false` so that pages with multiple SVGs could have a unique ID for each SVG that could be targeted for transitioning between light and dark mode.

Cheers,
Randy